spec/vcr/listings/applications_controller/new_sale.yml

```diff
@@ -236,127 +237,8 @@ http_interactions:
       Transfer-Encoding:
       - chunked
     body:
-      encoding: UTF-8
-      string: '{"access_token":"<<ACCESS_TOKEN>>","instance_url":"<<SALESFORCE_INSTANCE_URL>>","id":"https://<<SALESFORCE_HOST>>/id/00D0t0000000xVqEAI/005U00000066jl9IAA","token_type":"Bearer","issued_at":"1554761933753","signature":"GUz+svHcnwgQhezG+LBZVmKfjy5ib2Dt1dUIsXKbwjM="}'
+      encoding: ASCII-8BIT
+      string: '{"access_token":"<<ACCESS_TOKEN>>","instance_url":"<<SALESFORCE_INSTANCE_URL>>","id":"https://<<SALESFORCE_HOST>>/id/00D0t0000000xVqEAI/005U00000066jl9IAA","token_type":"Bearer","issued_at":"1567623395902","signature":"d0VTLlXF9o663HcnHCdKXLH0yWrUjVhtoqh9AZeTT1Y="}'
     http_version: 
-  recorded_at: Mon, 08 Apr 2019 22:18:53 GMT
-- request:
```

Comment:
@software-project looks like a lot of content went away here, any idea where this request went? just no longer needed because of updated test in a past PR?